AsyncDisplayKit.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |spec|
       'Base/*.{h,m}'
   ]
 
-  spec.frameworks = 'AssetsLibrary'
+  spec.frameworks = 'AssetsLibrary','MapKit'
   spec.weak_frameworks = 'Photos'
```

Comment:
Hey @appleguy  will investigate myself, hopefully we can change it to weak if it's not needed. I have time today from 16:30 GMT so will get back to you in time for 1.9.2, then we can make the getter/setter changes in a separate diff (probably tomorrow)